cumulus_library/builders/protected_table_builder.py

```diff
@@ -64,12 +67,22 @@ def prepare_queries(
                 TRANSACTION_COLS_TYPES,
             )
         )
-        if manifest._study_config.get("statistics_config"):
-            self.queries.append(
-                base_templates.get_ctas_empty_query(
-                    db_schema,
-                    statistics,
-                    STATISTICS_COLS,
-                    STATISTICS_COLS_TYPES,
-                )
-            )
+        files = manifest.get_file_list()
+        files = [file for file in files if file.endswith(".toml")]
+        if len(files) == 0:
+            return
```

Comment:
nit: this check is redundant - the loop will immediately end below if there are no files.

Reply:
what if you want to be :really: sure, though